src/main/java/org/commcare/formplayer/util/FormplayerDatadog.java

```diff
@@ -234,4 +237,11 @@ private String getTagValueToSend(String tagName, String tagValue) {
         }
     }
 
+    public static void handleKeepDropAPMTraces(boolean keepAPMTrace) {
```

Comment:
this only drops the traces after they have been collected, a more optimised way here might be to disable the trace collection itself. I looked around and found this (have not tested) - 

````
public static void disableTracing() {
        Tracer tracer = GlobalTracer.get();
        if (tracer instanceof datadog.opentracing.DDTracer) {
            ((datadog.opentracing.DDTracer) tracer).close();
        }
    }
````